rholang/tests/mercury-tut/convenience_methods_test.rho

```diff
@@ -0,0 +1,40 @@
+new testResult in {
```

Comment:
They are executed right now. I have changed CompilerTests behaviour [here](https://github.com/rchain/rchain/pull/1227/commits/e1646b9cca0d9653140183075c8474864d8f3a05) so that it executes every file under`tests` recursively.

Reply:
ah,sorry - didn't see that. Great then! :)